src/Components/ComUserTasksInfo.js

```diff
@@ -0,0 +1,27 @@
+export class ComUserTasksInfo extends Component {
+  constructor({
+    userId = 1, name = 'UserName', done = 1, inProgress = 1, waiting = 1
+  }) {
+    super();
+    this.userId = userId;
+    this.name = name;
+    this.done = done;
+    this.inProgress = inProgress;
+    this.waiting = waiting;
```

Comment:
why all these fields in `this`, but not in `this.state`? It's going to be changed (_done_ was 1, then it can become 2 etc)